internal_testsuites/golang/testsuite/startosis_request_wait_assert_test/starlark_validation_failure_invalid_port_id_request_test.go

```diff
@@ -0,0 +1,41 @@
+package startosis_request_wait_assert_test
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis-cli/golang_internal_testsuite/test_helpers"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+const (
+	requestInvalidPortIDTest       = "starlark-request-invalid-portid"
+	requestInvalidPortIDFailScript = `
+def run(plan):
+	service_config = ServiceConfig(
+		image = "mendhak/http-https-echo:26",
+		ports = {
+			"http-port": PortSpec(number = 8080, transport_protocol = "TCP")
+		}
+	)
+
+	plan.add_service(name = "web-server", config = service_config)
+	get_recipe = GetHttpRequestRecipe(
+		port_id = "invalid-port-id",
+		endpoint = "?input=foo/bar",
+		extract = {
+			"exploded-slash": ".query.input | split(\"/\") | .[1]"
+		}
+	)
+	response = plan.request(service_name = "web-server", recipe = get_recipe)
+`
+)
+
+func TestStarlark_InvalidPortIdRequest(t *testing.T) {
+	ctx := context.Background()
+	runResult := test_helpers.SetupSimpleEnclaveAndRunScript(t, ctx, requestInvalidPortIDTest, requestInvalidPortIDFailScript)
+
+	require.Nil(t, runResult.InterpretationError, "Unexpected interpretation error")
+	require.NotEmpty(t, runResult.ValidationErrors, "Expected validation errors")
+	require.Len(t, runResult.ValidationErrors, 1)
+	require.Contains(t, runResult.ValidationErrors[0].ErrorMessage, "Request required port ID 'invalid-port-id' to exist on service 'web-server' but it doesn't")
```

Comment:
Those tests make me sad. We're really testing a tiny feature, it doesn't quite fit in the idea I have of integration tests. We should think of a way to test our validation logic without having to write integration tests. Def not in the PR, more as food for thoughts.